src/App.js

```diff
@@ -1,54 +1,16 @@
 import React from "react";
-import "./App.scss";
-import smilies from "./assets/smilies.png";
-import i6 from "./assets/i6.png";
-import social from "./assets/social.png";
-import play from "./assets/play.png";
-import appStoreButton from "./assets/appstore_button.png";
-import menuIcon from "./assets/icons/menu_icon.png";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Login from "./pages/Login"
 
 function App() {
   return (
-    <div class="main-div">
-
-        <div class="left-container">
-
-          <div class="left-container__menu">
-            <img src={menuIcon} alt="menu_button"></img>
-          </div>
-
-          <div class="left-container__smilies" align="center">
-            <img src={smilies} alt="smilies"></img>
-          </div>
-
-          <p class="left-container__app-title" align="center">TARGET MVD</p>
-          <p class="left-container__app-subtitle" align="center">Find people near you & connect</p>
-          <p class="left-container__description-text" align="center">Create a  target wherever on the map, specify your interest: 
-            Travel, Dating, Music, etc and start conecting with others who share your interest.</p>
-
-        </div>
-
-        <div class="right-container" align="center">
-
-          <div class="right-container__i6-img">
-            <img src={i6} alt="i6"></img>
-          </div>
-
-          <div class="right-container__app-store-button">
-            <img src={appStoreButton} alt="app-store-button"></img>
-          </div>
-
-          <div class="right-container__social">
-            <img src={social} alt="social"></img>
-          </div>
-
-          <div class="right-container__play">
-            <img src={play} alt="play"></img>
-          </div>
-
-        </div>
-
-    </div>
+    <Router>
+        <Switch>
+          <Route path={["/", "/login"]}>
```

Comment:
This is fine just for now, but as soon as your app gets bigger I would recommend fetching this info from a `./routes.js` file ..., you can have a path component structure and add those here dynamically with the help of a map function

Reply:
I followed the steps in [create-app react page](https://create-react-app.dev/docs/adding-a-router/). Is there a doc to see a better way of doing this?

Reply:
I don't think there is a doc, I'm just talking about this code getting real big by adding multiple routes/components in the future. As I said, this is just fine for the current implementation, I just mentioned in case you want to investigate a little bit about it

Reply:
Ok I'll investigate about this when I add a new feature

Reply:
You can take a little sneak peek on our base project to better understand what I mean, but there is absolutely no rush on change this